packages/icons/src/apply-schema.svg

```diff
@@ -1,4 +1,4 @@
+<svg viewBox="0 0 31 37" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M6.74347 2.01355C6.92584 2.1973 6.92471 2.4941 6.74096 2.67646L6.11613 3.29658H11.7187C12.4954 3.29658 13.125 3.92618 13.125 4.70283V11.7187C13.125 12.4954 12.4954 13.125 11.7188 13.125H3.28125C2.5046 13.125 1.875 12.4954 1.875 11.7188V6.55974C1.875 6.30086 2.08487 6.09099 2.34375 6.09099C2.60263 6.09099 2.8125 6.30086 2.8125 6.55974V11.7188C2.8125 11.9776 3.02237 12.1875 3.28125 12.1875H11.7188C11.9776 12.1875 12.1875 11.9776 12.1875 11.7187V4.70283C12.1875 4.44395 11.9776 4.23408 11.7187 4.23408H6.11613L6.74096 4.8542C6.92471 5.03656 6.92584 5.33335 6.74347 5.5171C6.56111 5.70085 6.26432 5.70198 6.08056 5.51962L4.64818 4.09804C4.5595 4.01003 4.50963 3.89027 4.50962 3.76533C4.50962 3.64039 4.5595 3.52063 4.64818 3.43262L6.08056 2.01104C6.26432 1.82868 6.56111 1.8298 6.74347 2.01355Z" fill="#0278D5" stroke="#0278D5" stroke-width="0.5" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
Now we hard coding the colour. Can not consumer pass blue colour using prop? 
Right now icons may be supported with different back grounds and may be in red as well for handling error use cases. Hard coding colour will remove these supports.

Reply:
that's the reason currentColor was added earlier for the same icon. 

but looks like color is hardcoded in case of steps.